src/main/kotlin/no/nav/helsearbeidsgiver/bro/sykepenger/utils/ListUtils.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.helsearbeidsgiver.bro.sykepenger.utils
+
+/**
+ * Prioriterer `last` over `leading`.
+ * For en liste med størrelse 1 vil det eneste elementet ende i `last`, og `leading` blir tom.
+ *
+ * @return `null` dersom listen er tom.
+ */
+fun <T : Any> List<T>.leadingAndLast(): Pair<List<T>, T>? {
```

Comment:
Kan ikke dette forenkles ganske mye? Dette ser kanskje ikke like sofistikert ut, men det gjør samme nytte så vidt jeg skjønner (og er enklere å forstå - synes jeg..) : 

fun <T : Any> List<T>.leadingAndLast(): Pair<List<T>, T>? {
    if (isEmpty()) {
        return null
    }
    return if (size == 1) {
        Pair(emptyList<T>(), get(0))
    }
    else Pair(subList(0, size-1), get(size-1))
}
Da trenger du ikke partitionIndexed-funksjonen heller, så blir mindre kode tilsammen

Reply:
Det er unektelig mer rett frem, men jeg liker ikke så godt å sjekke om lister er tomme, for så å ha kode som baserer seg på det langt unna sjekken. Hvis det gir mening. Sliter litt med å formulere det, merker jeg.

Reply:
Men jeg fikk idé som kan gjøre det enklere. Gir det et forsøk før jeg tygger litt på tilbakemeldingen 🦷

Reply:
Hvordan synes du det ser ut om man bytter `partitionIndexed`-kallet med `chunked(size - 1)`?

Reply:
Jeg gjorde et forsøk på å gjøre det enklere, som ble en slags hybrid av ditt forslag og den eksisterende koden. Hva synes du nå?

Reply:
Det er så klart ikke like bra som mitt forslag ;) , men jeg godkjenner  :)
Mindre og lettere å forstå nå iaf 👍